websites/H/Hdfilmcehennemi/presence.ts

```diff
@@ -0,0 +1,55 @@
+const presence = new Presence({
+  clientId: "817292272508731392"
+}),
+pages: {[index: string]:string} = {
+  "/film-istekleri-yap/": "Film İstekleri",
+  "/site-hakkinda/": "Yasal Bildirim",
+  "/iletisim/": "İletişim"
+},
+categories: {[index: string]:string} = {
+  "/imdb-7-puan-uzeri-filmler/": "Imdb 7+ Filmler",
+  "/en-cok-yorumlananlar/": "En Çok Yorumlananlar",
+  "/en-cok-begenilen-filmler-izle/": "En Çok Beğenilenler",
+  "/dil/turkce-dublajli-film-izleyin/": "Türkçe Dublaj",
+  "/dil/turkce-altyazili-film-izleme-sitesi/": "Türkçe Altyazı"
+};
+
+presence.on("UpdateData", async () => {
+
+  let page = document.location.pathname;
+  const urlParams = new URLSearchParams(window.location.search),
+  myParam = urlParams.get('s');
+
+  if (!page.endsWith("/")) page = page + "/";
+
+  const presenceData: PresenceData = {
+    largeImageKey: "large"
+  };
+
+  if (pages[page]) {
+    presenceData.details = "Sayfada:";
+    presenceData.state = pages[page];
+  } else if (categories[page]) {
+    presenceData.details = "Film Arıyor:";
+    presenceData.state = categories[page];
+  } else if (page.includes("/category/") || page.includes("/tur/") || page.includes("/yil/")) {
+    presenceData.details = "Film Arıyor:";
+    let category = document.getElementsByClassName("tab-title")[0].firstChild.innerHTML;
+    category.includes("Kategorisi") ? category = category.replace(/Kategorisi(.*)/, "") : "";
+    category.includes("izle") ? category = category.replace(/izle(.*)/, "") : "";
+    presenceData.details = "Geziniyor:";
+    presenceData.state = category;
+  } else if (document.getElementById("player")) {
+    let movieName = document.getElementsByClassName("movie-title")[0].children[1].innerHTML;
+    movieName.includes("izle") ? movieName = movieName.replace(/izle(.*)/, "") : "";
+    presenceData.details = "Film İzliyor:";
+    presenceData.state = movieName;
```

Comment:
shouldnt it like showing playing/paused etc.?

Reply:
Site uses different players for movies (copyright reasons), so that's near impossible for me

Reply:
> Site uses different players for movies (copyright reasons), so that's near impossible for me

you should be able to get it using the video type and if in iframe using iframe.ts

Reply:
What I'm saying it looks like not every player is the same in that website. Doing this for every single one will be hard.

Reply:
> > Site uses different players for movies (copyright reasons), so that's near impossible for me
> 
> you should be able to get it using the video type and if in iframe using iframe.ts

I will try my best, but I have exams so I can do it after 2 weeks.

Reply:
you just need an update that will literally create an iframe.ts file, you could probably get away with copy pasting from a similar presence with a video player (i.e Kickassanime) provided the player has a `video` html tag

Reply:
> > > Site uses different players for movies (copyright reasons), so that's near impossible for me
> > 
> > 
> > you should be able to get it using the video type and if in iframe using iframe.ts
> 
> I will try my best, but I have exams so I can do it after 2 weeks.

will close the pr for now then, open a new one once you have added it.